lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -484,6 +484,7 @@ states:
     response:
       type: cri
       criId: nino
+      scope: identityCheck
```

Comment:
The NINO identity check which includes strength/validity scores shouldn't be needed for an HMRC KBV journey (and I believe scope is deprecated anyway so we would be using evidence_requested for this)